test/python/dagcircuit/test_dagdependency.py

```diff
@@ -24,7 +24,7 @@
 from qiskit.test import QiskitTestCase
 
 try:
-    import retworkx as rx
+    import reseaux as rx
 except ImportError:
     pass
```

Comment:
While we're at it, it'd be good to remove the `try/except` from this, since retworkx/reseaux is a core dependency.